wolox-books/src/store/modules/user.js

```diff
@@ -2,6 +2,9 @@ export const moduleUser = {
   state: {
     loginStatus: false
   },
+  getters: {
+    userStatus: state => state.loginStatus
```

Comment:
`userStatus` is not a really good name, try something like `isUserLogged` or something that explains that boolean better :smile: